default_parameters.py

```diff
@@ -60,7 +60,7 @@ def update_default_args(args):
 
         # Save update
 
-    args["save_every"] = 5000
+    args["save_every"] = 20
```

Comment:
20*64 << 5000 ?

Do we want to save that often ?

Reply:
outdated